src/admin/resolvers/mutations/ApprovedItem.ts

```diff
@@ -144,6 +145,18 @@ export async function updateApprovedItem(
     );
   }
 
+  // TODO: cleanly delete existintg authors as we do in collections
+  // get collectionStory internal id for deleting authors & checking for existing story
+  const existingItem = await getApprovedItemByUrl(context.db, data.url);
```

Comment:
this could be a new function `getApprovedItemByExternalId`? or do we think getting by URL (which is unique!) is fine?

Reply:
I opted for retrieving this via `getApprovedItemByExternalId`, the one that retrieves by URL is used to query when an item is added manually. I personally feel more comfortable using the id value rather than URL :).